control/controlclient/map.go

```diff
@@ -344,6 +346,9 @@ func (ms *mapSession) updateStateFromResponse(resp *tailcfg.MapResponse) {
 	}
 	if resp.Health != nil {
 		ms.lastHealth = resp.Health
+		warnings := expvar.Int{}
+		warnings.Set(int64(len(resp.Health)))
+		metricHealthMessages.Set(healthMessageLabel{Severity: "warning"}, &warnings)
```

Comment:
I think this should probably include health messages tracked by the client itself as well, reporting the number of errors returned by [health.OverallError](https://github.com/tailscale/tailscale/blob/f23932bd9858539ce4730b413709f4febb5bac93/health/health.go#L883).

(my mistake for writing down "health warnings sent by control" – I think I forgot about health messages tracked by the client itself).

Some health conditions are evaluated when that function is called, so I suspect we'd want varz to call it at scrape time.